app/controllers/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 require "report_exporter"
 
 class UsersController < Clearance::UsersController
-  before_action :ensure_correct_user, only: [:show, :edit, :update, :destroy, :applications, :delete_account]
+  before_action :ensure_correct_user, only: [:show, :edit, :update, :destroy, :applications, :confirm_delete, :delete_account]
```

Comment:
this is getting too long, we probably can use before_action :ensure_correct_user, except: [:new, :create] ?